R/pkg/R/functions.R

```diff
@@ -3652,3 +3652,56 @@ setMethod("posexplode",
             jc <- callJStatic("org.apache.spark.sql.functions", "posexplode", x@jc)
             column(jc)
           })
+
+#' create_array
+#'
+#' Creates a new array column. The input columns must all have the same data type.
+#'
+#' @param x Column to compute on
+#' @param ... other columns
```

Comment:
Should we adjust this for `concat(_ws)`, `least`, `greatest` and `countDistinct`?